cmd/kubelet/app/options/options.go

```diff
@@ -441,6 +433,9 @@ func AddKubeletConfigFlags(mainfs *pflag.FlagSet, c *kubeletconfig.KubeletConfig
 	fs.DurationVar(&c.Authorization.Webhook.CacheUnauthorizedTTL.Duration, "authorization-webhook-cache-unauthorized-ttl", c.Authorization.Webhook.CacheUnauthorizedTTL.Duration, ""+
 		"The duration to cache 'unauthorized' responses from the webhook authorizer.")
 
+	fs.StringVar(&c.CertDirectory, "cert-dir", c.CertDirectory, "The directory where the TLS certs are located. "+
+		"If tlsCertFile and tlsPrivateKeyFile are specified or corresponding kubelet config flags, this option will be ignored.")
```

Comment:
I think there is a mix up. I think we should keep the command line names if the "or corresponding kubelet config flags..." is used. Renaming was needed in the previous iteration when it was referring to the kubelet config flags.


```suggestion
		"If --tls-cert-file and --tls-private-key-file are specified or corresponding kubelet config flags set, this option will be ignored.")
```